.pylintrc

```diff
@@ -337,7 +337,7 @@ max-module-lines=1000
 # separator` is used to allow tabulation in dicts, etc.: {1  : 1,\n222: 2}.
 # `trailing-comma` allows a space between comma and closing bracket: (a, ).
 # `empty-line` allows space-only lines.
-no-space-check=trailing-comma,
+#no-space-check=trailing-comma,
```

Comment:
I do think it's a good idea for us to continue checking white-space. Would you mind re-enabling this check and fixing the code?

Do others have opinions about that?

Reply:
![image](https://user-images.githubusercontent.com/63213386/171921256-f8deb904-1488-42f4-a0cc-c0b446eae099.png)
The issue we're having with pylint is indicated in this image. Earlier documentation shows it's something vestigial and perhaps something that black maintains better than pylint. Ryan suggested this, but so far all I've been able to find are workarounds for this error as well.

Reply:
Huh. Weird. Okay. In that case, maybe let's get rid of the whole section since it isn't usable. Looks like this option was removed in v 2.6. 😬